internal/ids/ids.go

```diff
@@ -73,3 +73,7 @@ func SplitMailboxIDPairSlice(s []MailboxIDPair) ([]imap.InternalMailboxID, []ima
 
 	return internalMailboxIDs, mailboxIDs
 }
+
+const GluonRecoveryMailboxName = "RecoveredMessages"
```

Comment:
Why no space? `All Mail` has a space so can we have one for this too?

Reply:
Fixed.